pkg/sink/server/server.go

```diff
@@ -91,3 +92,61 @@ func (s *server) Start(ctx context.Context, inputOptions ...Option) {
 	defer log.Println("Successfully stopped the gRPC server")
 	grpcServer.GracefulStop()
 }
+
+// StartE starts the gRPC server via unix domain socket at configs.Addr and return error.
+func (s *server) StartE(ctx context.Context, inputOptions ...Option) error {
```

Comment:
I guess we don't need 2 similar functions for now, keeping `Start() error` is good enough.

Reply:
I updated the code